core/README.md

```diff
@@ -4,7 +4,7 @@ This crate contains core components used by Miden VM. These components include:
 * Miden VM instruction set, defined in the [Operation](/../main/core/src/operations/mod.rs) struct.
 * Miden VM program structure, defined in [Program](/../main/core/src/program/mod.rs) struct and described [here](https://0xpolygonmiden.github.io/miden-vm/design/programs.html).
 * Initial stack containner for Miden VM programs, defined in [StackInputs](/../main/core/src/inputs/stack.rs) struct.
-* Implementations of [advice sets](/../main/core/src/inputs/advice/mod.rs) which are used to provide nondeterministic inputs to the VM.
+* Implementations of [merkle sets](/../main/core/src/inputs/advice/mod.rs) which are used to provide nondeterministic inputs to the VM.
```

Comment:
I think this bullet-point should be removed, and we also might need to update some bullet points above.

Reply:
Done! I'm not sure what was that `main` so I just removed it. Maybe some leftover?

Reply:
Thanks! I do think we need to update the other bullet-points too - i.e., if we are mentioning `StackInputs` struct, we should probably mention `StackOutputs` struct as well. Similarly, for programs, I'd probably also mention `ProgramInfo` and `Kernel` structs.

Reply:
btw, I think `main` refers to the main branch - otherwise the links seem to be broken?

Reply:
Should we use `/../tree/next/` instead?

Reply:
I think it's OK to point to `main` so that we don't have to update when we merge `next` into `main`.

Reply:
Update: better keep as it was